bundles/paikkatietoikkuna/coordinatetransformation/components/CoordinateTable.js

```diff
@@ -172,12 +210,14 @@ Oskari.clazz.define('Oskari.coordinatetransformation.component.table', function(
             var cell;
             var rows = this.getElements().rows;
             for (var i = 0; i < rows.length; i++ ) {
-                jQuery(rows[i]).find("td").each(function(){
+                jQuery(rows[i]).find(".cellContent").each(function(){
                     cell = jQuery(this);
                     cell.text("");//empty();
-                    cell.removeClass("invalid-coord");
+                    cell.removeClass("invalid-coord"); //TODO row
                     if (isEditable){
                         cell.attr("contenteditable", true);
```

Comment:
You could remove the if/else and do:

    cell.attr("contenteditable", idEditable);